### PR TITLE
Bind vtgate to all interfaces not just localhost

### PR DESCRIPTION
**Pull request by arthurnn**
I am running vtcombo inside a Docker container and need to expose the
mysql protocol port to the host, so I can use it in my app.
Thus, binding mysql to localhost will not allow me to expose it to
the host.

### CODE REVIEW

**Comment by michael-berlin** [comment]
We cannot approve this change as is because you're changing the default here. Not binding to localhost is too broad because every time vtcombo is run (e.g. on an engineer's workstation who runs a unit test), a MySQL server is exposed and this may trip up automatic scanners in a network. On top of that, you may accidentally expose testing data this way.

Are you sure that the Docker port forwarding does not work if it's bound to localhost only? (You're probably right. I think, I recently ran into the same issue.)

To sum up: Making this configurable is fine, but not changing the default.

**Comment by arthurnn** [comment]
@michael-berlin sounds good. I will double check the Docker option. Otherwise, I will make this a configuration instead.

**Comment by sougou** [comment]
@arthurnn Are you going to rework this PR, or should this be closed?

**Comment by arthurnn** [comment]
@sougou will re-work it today

**Comment by arthurnn** [comment]
Sorry for the 1-month delay. But, I got this and updated the code to add a config now with the address to bind the vtcombo MySQL port to.
Thoughts?

**Comment by sougou** [comment]
This looks good. Can you pull in the latest changes from master where @alainjobart fixed some localhost issues? I want to be sure the tests pass after that.

**Comment by arthurnn** [comment]
@sougou all done

**Comment by sougou** [comment]
LGTM
